vendor/k8s.io/kubernetes/staging/src/k8s.io/client-go/transport/cache.go

```diff
@@ -67,9 +67,9 @@ func (c *tlsTransportCache) get(config *Config) (http.RoundTripper, error) {
 	defer c.mu.Unlock()
 
 	// See if we already have a custom transport for this config
-	if t, ok := c.transports[key]; ok {
-		return t, nil
-	}
+	//if t, ok := c.transports[key]; ok {
```

Comment:
using the transport cache results in a create project call that when run through the aggregator hangs until the binary build resthandler returns